packages/schematics/src/collection/ngrx/files/__directory__/__fileName__.facade.ts__tmpl__

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+
+import { Store } from '@ngrx/store';
+
+import { <%= className %>State } from './<%= fileName %>.reducer';
+import { <%= propertyName %>Query } from './<%= fileName %>.selectors';
+import { Load<%= className %> } from './<%= fileName %>.actions';
+
+@Injectable({
+  providedIn : 'root'
```

Comment:
Should it be 'root', or should it be the module defining the facade? I imagine the latter.

Reply:
good catch. I will remove this.